04 - Guides, Workflows, & Courses/Courses/🗂️ Courses.md

```diff
@@ -12,7 +12,3 @@ publish: true
 
 #placeholder/description 
 
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```

Comment:
I'd keep this one as a manual MOC

Reply:
Thanks.

Done: https://github.com/obsidian-community/obsidian-hub/commit/2ea8a007691a5c204e48c2af057f933c6f636815